parcels/interaction/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .baseinteractionkernel import BaseInteractionKernel  # noqa
+from .interactionkernelsoa import InteractionKernelSOA  # noqa
+from .hash_spherical import HashSphericalNeighborSearch  # noqa
+from .brute_force import BruteFlatNeighborSearch  # noqa
```

Comment:
Now, when I see this, I see that the interaction only works with SoA - as previously agreed. That's nice. For further development purposes, it may be nice to generally add (in the docstring of the member methods of the SoA structures) a not saying which functions is required to be defined for InteractionKernels to work. This comment mainly extends to `ParticleSet` and `Kernel` (as, the way I see it, particle-particle interaction has no impact of `ParticleFile` write-outs).